parm/atm/obs/config/iasi_metop-b.yaml

```diff
@@ -0,0 +1,756 @@
+            3354, 3366, 3375, 3378, 3411, 3416, 3432, 3438, 3440, 3442, 3444, 3446,
+            3448, 3450, 3452, 3454, 3458, 3467, 3476, 3484, 3491, 3497, 3499, 3504,
+            3506, 3509, 3518, 3527, 3555, 3575, 3577, 3580, 3582, 3586, 3589, 3599,
+            3610, 3626, 3638, 3646, 3653, 3658, 3661, 3673, 3689, 3700, 3710, 3726,
+            3763, 3814, 3841, 3888, 4032, 4059, 4068, 4082, 4095, 4160, 4234, 4257,
+            4411, 4498, 4520, 4552, 4567, 4608, 4646, 4698, 4808, 4849, 4920, 4939,
+            4947, 4967, 4991, 4996, 5015, 5028, 5056, 5128, 5130, 5144, 5170, 5178,
+            5183, 5188, 5191, 5368, 5371, 5379, 5381, 5383, 5397, 5399, 5401, 5403,
+            5405, 5446, 5455, 5472, 5480, 5483, 5485, 5492, 5497, 5502, 5507, 5509,
+            5517, 5528, 5558, 5697, 5714, 5749, 5766, 5785, 5798, 5799, 5801, 5817,
+            5833, 5834, 5836, 5849, 5851, 5852, 5865, 5869, 5881, 5884, 5897, 5900,
+            5916, 5932, 5948, 5963, 5968, 5978, 5988, 5992, 5994, 5997, 6003, 6008,
+            6023, 6026, 6039, 6053, 6056, 6067, 6071, 6082, 6085, 6098, 6112, 6126,
+            6135, 6140, 6149, 6154, 6158, 6161, 6168, 6174, 6182, 6187, 6205, 6209,
+            6213, 6317, 6339, 6342, 6366, 6381, 6391, 6489, 6962, 6966, 6970, 6975,
+            6977, 6982, 6985, 6987, 6989, 6991, 6993, 6995, 6997, 6999, 7000, 7004,
+            7008, 7013, 7016, 7021, 7024, 7027, 7029, 7032, 7038, 7043, 7046, 7049,
+            7069, 7072, 7076, 7081, 7084, 7089, 7099, 7209, 7222, 7231, 7235, 7247,
+            7267, 7269, 7284, 7389, 7419, 7423, 7424, 7426, 7428, 7431, 7436, 7444,
+            7475, 7549, 7584, 7665, 7666, 7831, 7836, 7853, 7865, 7885, 7888, 7912,
+            7950, 7972, 7980, 7995, 8007, 8015, 8055, 8078
+
+obs operator:
+  name: CRTM
+  Absorbers: [H2O,O3,CO2]
+  obs options:
+    Sensor_ID: iasi_metop-b 
+    EndianType: little_endian
+    CoefficientPath: $(DATA)/crtm/
+  linear obs operator:
+    Absorbers: [H2O,O3]
+
+obs bias:
+  input file: $(DATA)/obs/$(GPREFIX)iasi_metop-b.satbias.nc4
+  output file: $(DATA)/bc/$(APREFIX)iasi_metop-b.satbias.nc4
+  variational bc:
+    predictors:
+    - name: constant
+    - name: lapse_rate
+      order: 2
+      tlapse: &iasi_metop-b_tlapse $(DATA)/obs/$(GPREFIX)iasi_metop-b.tlapse.txt
+    - name: lapse_rate
+      tlapse: *iasi_metop-b_tlapse
+    - name: emissivity
+    - name: scan_angle
+      order: 4
+    - name: scan_angle
+      order: 3
+    - name: scan_angle
+      order: 2
+    - name: scan_angle
+  covariance:
+    minimal required obs number: 20
+    variance range: [1.0e-6, 10.0]
+    step size: 1.0e-4
+    largest analysis variance: 10000.0
+    prior:
+      input file: $(DATA)/obs/$(GPREFIX)iasi_metop-b.satbias_cov.nc4
+      inflation:
+        ratio: 1.1
+        ratio for small dataset: 2.0
+    output file: $(DATA)/bc/$(APREFIX)iasi_metop-b.satbias_cov.nc4
+
+obs pre filters:
+# Step 0-A: Create Diagnostic Flags
```

Comment:
Previously this filter was **not** commented out in `iasi_metop-b.yaml`.   This filters are not commented out in `iasi_metop-a.yaml`.  Why?

Reply:
> Previously this filter was **not** commented out in `iasi_metop-b.yaml`. This filters are not commented out in `iasi_metop-a.yaml`. Why?

@RussTreadon-NOAA Thanks for spotting this.  Turning off `DiagnistocFlags`should be done for both IASI platforms to save memory. I just updated the iasi_metop-a YAML.